numpy/polynomial/chebyshev.py

```diff
@@ -2059,6 +2059,6 @@ def interpolate(cls, func, deg, domain=None, args=()):
 
     # Virtual properties
     nickname = 'cheb'
-    domain = np.array(chebdomain)
-    window = np.array(chebdomain)
+    _domain = np.array(chebdomain)
+    _window = np.array(chebdomain)
```

Comment:
A little alarming that these are mutable.

Reply:
Do you mean that the original (red) is alarming or the proposed change? I had figured the prepended-`_`-means-"private" convention was enough to discourage any user from changing the default class variables.